apps/angular/projection/src/app/ui/card/card-row.directive.ts

```diff
@@ -0,0 +1,4 @@
+import { Directive } from '@angular/core';
+
+@Directive({ standalone: true, selector: 'ng-template[cardRow]' })
+export class CardRowDirective {}
```

Comment:
you can go one step further and add the type checking. 
See https://angular-challenges.vercel.app/challenges/angular/4-context-outlet-typed/

Reply:
I saw that in your solution and didn't really understand it. I'll come back to this after I do challenge 4.

Reply:
yes that's not very common but very useful, because currently when you call your template outlet, your object is of type any.